### PR TITLE
Add stackdriver configs

### PR DESCRIPTION
**Pull request by richardwxn**
Continual effort of previous closed PR: https://github.com/istio/tools/pull/686/. Update based on new framework.

### CODE REVIEW

**Comment by richardwxn** [comment]
put a hold for mege now until the next framework succeed in the pipeline

File: perf/benchmark/configs/istio/telemetryv2_stats/cpu_mem.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
-# Configuration Set5: CPU and memory with telemetryv2 using NullVM.
-telemetry_mode: "v2-nullvm"
+# Configuration Set5: CPU and memory with telemetryv2 using NullVM with stats envoyfilter.
+telemetry_mode: "v2-stats-nullvm"
```
Comment (carolynhu) [inline perf/benchmark/configs/istio/telemetryv2_stats/cpu_mem.yaml]: only has stats filter, I though this have both stats and metadata exchange filters?

Comment (richardwxn) [reply perf/benchmark/configs/istio/telemetryv2_stats/cpu_mem.yaml]: metadata-exchange is required for both cases by now, we may just need to distinguish between prometheus and stackdriver

Comment (mandarjog) [reply perf/benchmark/configs/istio/telemetryv2_stats/cpu_mem.yaml]: v2 stats should have mx too. It is important to benchmark the actual configuration that users use.

**Pull request merged**